### PR TITLE
Throw arrivals dumps randomly

### PR DESCRIPTION
**Pull request by metalgearsloth**
Stops people using it for free spacing.

### CODE REVIEW

File: Content.Server/Shuttles/Systems/ArrivalsSystem.cs [hunk 1]
```diff
@@ -42,6 +43,15 @@ public sealed class ArrivalsSystem : EntitySystem
...
     [Dependency] private readonly StationSystem _station = default!;
+    [Dependency] private readonly ThrowingSystem _throw = default!;
+
+    /// <summary>
+    /// Throw any entities that get dumped.
+    /// </summary>
+    /// <remarks>
+    /// Stops people using the shuttle as a free spacing.
+    /// </remarks>
+    public const bool ThrowOnDump = true;
```
Comment (mirrorcult) [inline Content.Server/Shuttles/Systems/ArrivalsSystem.cs:54]: cvar?

**Comment by AJCM-git** [comment]
Wouldnt this make it worse since instead of being spaced close to the station you can be thrown into deepspace in maps where arrivals isn't closed in

**Comment by metalgearsloth** [comment]
> Wouldnt this make it worse since instead of being spaced close to the station you can be thrown into deepspace in maps where arrivals isn't closed in

So the thing is currently people go into the arrivals shuttle as a free way to get outside the station without externals access / hacking so this is more as a punishment for those people.

**Comment by AJCM-git** [comment]
> So the thing is currently people go into the arrivals shuttle as a free way to get outside the station without externals access / hacking so this is more as a punishment for those people.

seems like a very specific thing that should be fixed other ways because this just punishes even more confused new players that re-enter or stay in the shuttle

**Comment by metalgearsloth** [comment]
https://github.com/space-wizards/space-station-14/issues/16419 was option B.

**Pull request closed without merging**